apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/RowFilters.cs

```diff
@@ -0,0 +1,380 @@
+        /// </param>
+        /// <param name="endTimestamp">
+        /// Of the range of versions from which cells should be matched, exclusive upper bound timestamp
+        /// whose milliseconds since the Unix epoch should be used as the version value. If null, interpreted
+        /// as infinity.  It must be specified in UTC.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter TimestampRange(DateTime? startTimestamp, DateTime? endTimestamp) =>
+            VersionRange(new BigtableVersionRange(startTimestamp, endTimestamp));
+
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which matches only cells
+        /// with values that fall within the given range.
+        /// </summary>
+        /// <param name="range">
+        /// The range of values to match. Must not be null.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter ValueRange(ValueRange range) =>
+            new RowFilter { ValueRangeFilter = GaxPreconditions.CheckNotNull(range, nameof(range)) };
+
+        // TODO: Add additional ValueRange... methods to take start/end values?
+
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which matches only cells
+        /// with values that satisfy the given regular expression.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// Note that, since cell values can contain arbitrary bytes, the `\C` escape
+        /// sequence must be used if a true wildcard is desired. The `.` character
+        /// will not match the new line character `\n`, which may be present in a
+        /// binary value.
+        /// </para>
+        /// <para>
+        /// Note that string is implicitly convertible to <see cref="BigtableByteString"/>,
+        /// so <paramref name="regex"/> can be specified using a string as well and its UTF-8
+        /// representations will be used.
+        /// </para>
+        /// </remarks>
+        /// <param name="regex">
+        /// The RE2 regex used to match cell values.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter ValueRegex(BigtableByteString regex) =>
+            new RowFilter { ValueRegexFilter = regex.Value };
+
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which matches only cells
+        /// with versions within the given range.
+        /// </summary>
+        /// <param name="range">
+        /// The range of versions from which cells should be matched. Must not be null.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter VersionRange(BigtableVersionRange range) =>
+            new RowFilter { TimestampRangeFilter = GaxPreconditions.CheckNotNull(range, nameof(range)).ToTimestampRange() };
+
+        /// <summary>
```

Comment:
Possibly remove this overload for now and replace it with a TODO, to match the others ranges?

Reply:
Done, but I'm going to leave `TimestampRange` since that's the name used in the docs, even though it delegates to `VersionRange`.